lib/validate.js

```diff
@@ -18,6 +18,9 @@ module.exports = {
         throw new this.serverless.classes
           .Error('The webpack plugin could not find the configuration file at: ' + webpackConfigFilePath);
       }
+      if (this.options.function) {
+        process.env.serverlessFunction = this.options.function;
```

Comment:
@HyperBrain I'm using this env var with `webpack.config.js`. But reading further comments I realized that you guys have suggested a better approach.